docsrc/sasl/options.rst

```diff
@@ -96,6 +96,30 @@ GSSAPI
 
    Default: /etc/krb5.keytab (system dependant)
 
+.. option:: ccache [<path>]

```

Comment:
The term cache is exceedingly vague, I suggest something like client_ccache or delegated_ccache_store or similar ...

Reply:
Settled for ccache_store instead. It's similar enough to other systems I've seen.